src/index.html

```diff
@@ -9,10 +9,53 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="css/style.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic"
+      rel="stylesheet"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <main class="main page__main">
+      <section
+        class="product-card"
+        data-qa="card"
+      >
+        <div class="product-card__container">
+          <div class="product-card__image">
+            <img
+              src="./images/imac.jpeg"
+              alt="Card Image"
+            />
+          </div>
+          <h1 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
```

Comment:
```suggestion
          <h2 class="product-card__title">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h2>
```
The h1 tag should be single on the page. In this case, it's expected to have multiple cards on the page